README.md

```diff
@@ -34,146 +38,106 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
-   ```
+#### Provision Storage
+Before Rook can start provisioning storage, a StorageClass needs to be created. This is used to specify the storage privisioner, parameters, admin secret and other information needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.
+Rook already creates a default admin and demo user, whose secrets are already specified in the sample [rook-storageclass.yaml](demo/kubernetes/rook-storageclass.yaml).
 
-4. Use an S3 compatible client to create a bucket in the object store
+However, before we proceed, we need to specify the Ceph monitor endpoints. You can find them by running this line (you will need `jq`). This will generated a comma-separated list of monitor IPs and ports `6790` and place it into the `monitors` param of the `rook-storageclass.yaml`
 
-   ```bash
-   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ export MONS=$(kubectl -n rook get pod mon0 mon1 mon2 -o json|jq ".items[].status.podIP"|tr -d "\""|sed -e 's/$/:6790/'|paste -s -d, -)
+$ sed 's#INSERT_HERE#'$MONS'#' rook-storageclass.yaml | kubectl create -f -
+``` 
 
-5. List all buckets in the object store
+#### Consume the storage
 
-   ```bash
-   s3cmd ls --no-ssl --host=${AWS_ENDPOINT} --host-bucket=
-   ```
+Now that rook is running and integrated with Kubernetes, we can create a sample app to consume the block storaged provisioned by rook. We will create the classic wordpress and mysql apps.
+Both these apps will make use of block volumes provisioned by rook.
 
-6. Upload a file to the newly created bucket
+Start mysql and wordpress:
 
-   ```bash
-   echo "Hello Rook!" > /tmp/rookObj
-   s3cmd put /tmp/rookObj --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
-   ```
+```
+$ kubectl create -f mysql.yaml
+$ kubectl create -f wordpress.yaml
+```
 
-7. Download and verify the file from the bucket
+Both of these apps create a block volume and mount it to their respective pod. You can see the Kubernetes volume claims by running the following:
 
-   ```bash
-   s3cmd get s3://rookbucket/rookObj /tmp/rookObj-download --no-ssl --host=${AWS_ENDPOINT} --host-bucket=
-   cat /tmp/rookObj-download
-   ```
+```
+$ kubectl get pvc
+NAME             STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+mysql-pv-claim   Bound     pvc-95402dbc-efc0-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
+wp-pv-claim      Bound     pvc-39e43169-efc1-11e6-bc9a-0cc47a3459ee   20Gi       RWO           1m
+```
 
-### Kubernetes
+Get the cluster IP of the wordpress app and enter it in your brower:
 
-To run a Kubernetes cluster with Rook for persistent storage go [here](https://github.com/rook/coreos-kubernetes)
+```
+$ kubectl get svc wordpress
+NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+wordpress   10.3.0.155   <pending>     80:30841/TCP   2m
+```
 
-### CoreOS
+You should see the wordpress app running.
 
```

Comment:
we may want to provide a script in the demo folder that cleans up all the rook and demo app k8s artifacts and point them to it at the end of the demo